tests/api/core/content-manager/search.test.api.js

```diff
@@ -191,10 +191,9 @@ describe('Search query', () => {
 
       expect(Array.isArray(res.body.results)).toBe(true);
       expect(res.body.results.length).toBe(data.beds.length);
-      // TODO V5: Filter out i18n fields if content type is not localized
-      expect(res.body.results.map(omit([...CREATOR_FIELDS, 'localizations', 'status']))).toEqual(
-        expect.arrayContaining(data.beds)
-      );
+      expect(
```

Comment:
Sorry to be annoying on this, but could we add a test for if i18n is enabled on roles and see that the fields still show up? That's the one case where this would potentially become a breaking change, since we have our own custom sanitization on this I could see someone potentially breaking it by accident in the future.

Reply:
How do you want to enable i18n on the roles content type though? Or are you talking about the relation? :thinking:

Reply:
My bad, I thought we let it be customized like the User ct. It looks like we don't allow it, so it's fine, someone would have to be messing with things they shouldn't